lib/doc/worksheet.js

```diff
@@ -806,6 +809,17 @@ class Worksheet {
     return Object.values(this.tables);
   }
 
+  // =========================================================================
+  // Pivot Tables
+  addPivotTable(model) {
+    const pivotTable = makePivotTable(this, model);
+
+    this.pivotTables.push(pivotTable);
+    this.workbook.pivotTables.push(pivotTable);
```

Comment:
What will happen if someone remove the pivot table from one of these places, or overwrite one of them?
What do you think about making `wb.workbook.pivotTables` a   getter returning pivottables from all worksheets?  Or .. a function?
Can it attach one `pivotTable` to multiple sheets?

Reply:
> What will happen if someone remove the pivot table from one of these places, or overwrite one of them?

@Siemienik It seems you're pointing at the mutability of `worksheet.pivotTables` and `worksheet.workbook.pivotTables` and concerns for that clients might mutate this. If you think this type of safety is needed and consistent with elsewhere in ExcelJS, we can definitely make it a getter. I'll take your advice here.

> Can it attach one pivotTable to multiple sheets?

At this time, I intend for there to be at most one pivotTable in at most one sheet across the entire workbook. The user should not have to mutate `worksheet.pivotTables` or `worksheet.workbook.pivotTables` and only access `worksheet.addPivotTable`.